Dockerfile

```diff
@@ -20,6 +20,14 @@ RUN . $VENV_DIR/bin/activate && pip install --no-cache-dir -r requirements.txt
 # CVE-2022-40897
 RUN . $VENV_DIR/bin/activate && pip install setuptools==65.5.1
 
+# Azure Dedicated SQL Pools uses pyodbc which requires unixODBC and 'ODBC Driver 17 for SQL Server'
+RUN apt-get update \
+    && apt-get install -y gnupg gnupg2 gnupg1 curl apt-transport-https \
+    && curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
+    && curl https://packages.microsoft.com/config/debian/10/prod.list > /etc/apt/sources.list.d/mssql-release.list \
+    && apt-get update \
+    && ACCEPT_EULA=Y apt-get install -y msodbcsql17 unixodbc unixodbc-dev
+
```

Comment:
you need to install the same packages using `yum` in the `lambda` image at the end of this same `Dockerfile` and test this with a lambda agent

Reply:
added and then tested with test-lambda-agent account's non-vpc collector and it worked
<img width="857" alt="Screenshot 2023-12-01 at 2 21 27 PM" src="https://github.com/monte-carlo-data/apollo-agent/assets/97194436/ef68b3a9-59c4-473a-af97-a546105666d5">